src/NotebookExperiments.ts

```diff
@@ -30,10 +30,11 @@ export class NotebookExperimentWidget extends Widget {
     tableHead.className = "dlw-Table-experiments";
 
     let tableHeadRow = tableHead.insertRow();
-    tableHeadRow.insertCell(0).innerText = "Model ID";
-    tableHeadRow.insertCell(1).innerText = "Name";
-    tableHeadRow.insertCell(2).innerText = "Description";
-    tableHeadRow.insertCell(3).innerText = "Status";
+    tableHeadRow.className = "dlw-Table-experiments-header";
+    tableHeadRow.insertCell(0).innerText = "MODEL ID";
```

Comment:
Is capital letters better than camel case in this scenario?

Reply:
This is a suggestion to make it consistent with the most of the title styles in jupyter side bar menus